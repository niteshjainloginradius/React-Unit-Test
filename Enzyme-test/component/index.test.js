import React from 'react';
//var enzyme = require('enzyme');
//var Adapter = require('enzyme-adapter-react-16');
import Welcome from './../../src/index';
import renderer from 'react-test-renderer';
//var jest =require('jest');
//enzyme.configure({ adapter: new Adapter() });
test('renders correctly', () => {
  const tree = renderer.create(<Welcome/>).toJSON();
  expect(tree).toMatchSnapshot();
  })