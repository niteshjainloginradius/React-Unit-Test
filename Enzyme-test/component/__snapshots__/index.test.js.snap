// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div>
    <link
      crossOrigin="anonymous"
      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
      rel="stylesheet"
    />
    <div>
      <nav
        className="navbar navbar-expand navbar-dark bg-dark"
      >
        <a
          className="navbar-brand"
          href="#home"
        >
          Navbar
        </a>
        <div
          className="mr-auto navbar-nav"
          onKeyDown={[Function]}
        >
          <a
            className="nav-link"
            data-rb-event-key="#home"
            href="#home"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Home
          </a>
          <a
            className="nav-link"
            data-rb-event-key="#features"
            href="#features"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Features
          </a>
          <a
            className="nav-link"
            data-rb-event-key="#pricing"
            href="#pricing"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Pricing
          </a>
        </div>
        <form
          className="form-inline"
        >
          <input
            className="mr-sm-2 form-control"
            placeholder="Search"
            type="text"
          />
          <button
            className="btn btn-outline-info"
            disabled={false}
            type="button"
          >
            Search
          </button>
        </form>
      </nav>
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-4"
      >
        <div>
          <form
            className=""
          >
            <div
              className="form-group row"
            >
              <label
                className="form-label col-form-label col-sm-2"
              >
                Email
              </label>
              <div
                className="col-sm-10"
              >
                <input
                  className="form-control"
                  id="formHorizontalEmail"
                  name="email"
                  onChange={[Function]}
                  placeholder="Email"
                  type="email"
                />
              </div>
            </div>
            <div
              className="form-group row"
            >
              <label
                className="form-label col-form-label col-sm-2"
              >
                Password
              </label>
              <div
                className="col-sm-10"
              >
                <input
                  className="form-control"
                  id="formHorizontalPassword"
                  name="password"
                  onChange={[Function]}
                  placeholder="Password"
                  type="password"
                />
              </div>
            </div>
            <div
              className="form-group row"
            >
              <div
                className="col-sm-10 offset-sm-2"
              >
                <div
                  className="form-check"
                >
                  <input
                    className="form-check-input"
                    disabled={false}
                    id="formHorizontalCheck"
                    name="remember"
                    onChange={[Function]}
                    type="checkbox"
                    value={false}
                  />
                  <label
                    className="form-check-label"
                    htmlFor="formHorizontalCheck"
                    title=""
                    type="checkbox"
                  >
                    Remember me
                  </label>
                </div>
              </div>
            </div>
            <div
              className="form-group row"
            >
              <div
                className="col-sm-10 offset-sm-2"
              >
                <button
                  className="btn btn-primary"
                  controlId="buttonSign"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Sign in
                </button>
              </div>
            </div>
          </form>
          <div
            id="childrean"
          >
            <div
              id="child"
            >
              
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-md-7"
      >
        <div
          className="row"
        >
          <div
            className="col-md-4 card bg-primary text-white"
          >
            <div
              className="card-header"
            >
              Header
            </div>
            <div
              className="card-body"
            >
              <div
                className="card-title h5"
              >
                Primary Card Title
              </div>
              <p
                className="card-text"
              >
                Some quick example text to build on the card title and make up the bulk of the card's content.
              </p>
            </div>
          </div>
          <div
            className="col-md-4 card bg-secondary text-white"
          >
            <div
              className="card-header"
            >
              Header
            </div>
            <div
              className="card-body"
            >
              <div
                className="card-title h5"
              >
                Secondary Card Title
              </div>
              <p
                className="card-text"
              >
                Some quick example text to build on the card title and make up the bulk of the card's content.
              </p>
            </div>
          </div>
          <div
            className="col-md-4 card bg-success text-white"
          >
            <div
              className="card-header"
            >
              Header
            </div>
            <div
              className="card-body"
            >
              <div
                className="card-title h5"
              >
                Success Card Title
              </div>
              <p
                className="card-text"
              >
                Some quick example text to build on the card title and make up the bulk of the card's content.
              </p>
            </div>
          </div>
          <div
            className="col-md-4 card bg-danger text-white"
          >
            <div
              className="card-header"
            >
              Header
            </div>
            <div
              className="card-body"
            >
              <div
                className="card-title h5"
              >
                Danger Card Title
              </div>
              <p
                className="card-text"
              >
                Some quick example text to build on the card title and make up the bulk of the card's content.
              </p>
            </div>
          </div>
          <div
            className="col-md-4 card bg-warning text-white"
          >
            <div
              className="card-header"
            >
              Header
            </div>
            <div
              className="card-body"
            >
              <div
                className="card-title h5"
              >
                Warning Card Title
              </div>
              <p
                className="card-text"
              >
                Some quick example text to build on the card title and make up the bulk of the card's content.
              </p>
            </div>
          </div>
          <div
            className="col-md-4 card bg-info text-white"
          >
            <div
              className="card-header"
            >
              Header
            </div>
            <div
              className="card-body"
            >
              <div
                className="card-title h5"
              >
                Info Card Title
              </div>
              <p
                className="card-text"
              >
                Some quick example text to build on the card title and make up the bulk of the card's content.
              </p>
            </div>
          </div>
          <div
            className="col-md-4 card bg-dark text-white"
          >
            <div
              className="card-header"
            >
              Header
            </div>
            <div
              className="card-body"
            >
              <div
                className="card-title h5"
              >
                Dark Card Title
              </div>
              <p
                className="card-text"
              >
                Some quick example text to build on the card title and make up the bulk of the card's content.
              </p>
            </div>
          </div>
          <div
            className="col-md-4 card bg-light"
          >
            <div
              className="card-header"
            >
              Header
            </div>
            <div
              className="card-body"
            >
              <div
                className="card-title h5"
              >
                Light Card Title
              </div>
              <p
                className="card-text"
              >
                Some quick example text to build on the card title and make up the bulk of the card's content.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div>
      <h1>
        Footer
      </h1>
    </div>
  </div>
</div>
`;
